### ISSUE TITLE
Re-sync date from Gravity form to already created Custom Post

### ISSUE THREAD

**Issue by snowshadows** [pods-framework/pods-gravity-forms#105]
## Expected Behavior

I am using Pods Custom Content Types and Fields and Gravity Forms Add-on. When the user fills out a Gravity form, the information is fed to the Custom Post Type and populates it, and saves the post as “pending review.”I have recently connected some items from the form entry to custom fields on the custom post. I would like it to push that information to the already created, but pending, posts. 

## Possible Solution
It would be nice to be able to have the custom post access the originall form entry and effectively "sync" the information. This would not be good to do indefinitely, but only when requested.  

## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Form Upload

<details>
```


```
</details>

## Pods Package Export

<details>
```

```
</details>

## Todos
- [ ] Tests
- [ ] Documentation

## WordPress Environment
<details>
```
Copy and paste your System Details from **Pods Admin > Settings > Debug Information** in WordPress admin here.
```
</details>

**Comment by snowshadows** [comment]
I am still not sure where to put this feature request.

**Comment by snowshadows** [comment]
Hello?

**Comment by jimtrue** [comment]
This is a Feature Request, not a bug. Since we're a free plugin, we have to balance our workload over higher overall feature hitting items, which include bug requests and making sure all of our fields properly map to Gravity Form fields.

We only just added the ability to edit the existing post with a Gravity Form and it still has some issues around properly connecting those two items. Before we'd ever be able to perform a 'sync' from a form, we'd have to rough out those issues first, including adding some sort of structure that locked an 'entry' to an actual 'post'.

**Comment by snowshadows** [comment]
Thank you for getting back. I'm pretty sure I did add it to the feature requests, but I do appreciate your response here. Are you aware of any issues created by upgrading to Gravity Forms 2.3.1, in general?

**Comment by jimtrue** [comment]
Yes and we have a bug report on it (#108) and a patch in the works, which you can download from here to test (https://github.com/pods-framework/pods-gravity-forms/tree/1.4-beta). Most people had issues with the 2.3 upgrade, not just those that were using Pods because it included a major Database change.

**Comment by snowshadows** [comment]
Thank you.

**Comment by sc0ttkclark** [comment]
Fixed in #86 (ZIP: https://github.com/pods-framework/pods-gravity-forms/archive/feature/skc-fixes.zip)

We will have documentation for our syncing (via WP-CLI) in our release post.
